git/homework.py

```diff
@@ -78,8 +103,15 @@ def multiple_ints_with_conversion(first_value: Any, second_value: Any) -> int:
             print("Not valid input data")
         >>> "Not valid input data"
     """
-    pass
-
+    try:
+        first_value1=int(first_value)
+        second_value1=int(second_value)
+        mult=first_value1*second_value1
```

Comment:
тут можна уникнути зайвого переприсвоєння:
```suggestion
        return first_value1 * second_value1
```